environments/staging/terraform.yml

```diff
@@ -194,11 +194,9 @@ servers:
     network_tier: "db-private"
     az: "b"
     volume_size: 30
-    volume_type: "gp3"
     volume_encrypted: yes
     block_device:
       volume_size: 80
-      volume_type: "gp3"
```

Comment:
I think this is an unrelated change. Maybe put this in a different PR if you still want to make it?

Reply:
yes, Danny. Sorry for pushing this change. Initially, it should lot of files are changes during that time I have missed this. Will fix this in the next commits.